test/cases/api/common/common.sh

```diff
@@ -19,8 +19,20 @@ function _instanceCheck() {
   echo "✔️ Instance checking"
   local _ssh="$1"
 
-  # Check if postgres is installed
-  $_ssh rpm -q postgresql dummy
+  # Retry loop to wait for instance to be ready
+  # This is here especially because of gcp test
+  RETRIES=10
+  for i in $(seq 1 $RETRIES); do
+    echo "Attempt $i of $RETRIES: Checking instance status..."
+    # Check if postgres is installed
+    if eval "$_ssh rpm -q postgresql dummy"; then
```

Comment:
Would it make sense to use just something like `"$_ssh true"` instead? This could fail also if the packages are not installed and it would retry multiple times, no?

Reply:
Riiight, I didn't think of that, I was happy to have something that works after trying out the different variations of `gcloud` ssh flags while fighting with bash :sweat_smile: ... I'll update it, thanks!